rundeckapp/grails-app/views/framework/_pluginConfigPropertyFormField.gsp

```diff
@@ -332,7 +332,9 @@
         %{-- selector for accessible storage --}%
         <g:set var="storageRoot" value="${prop.renderingOptions?.(StringRenderingConstants.STORAGE_PATH_ROOT_KEY)?:'/'}"/>
         <g:set var="storageFilter" value="${prop.renderingOptions?.(StringRenderingConstants.STORAGE_FILE_META_FILTER_KEY)?:''}"/>
-        <div class="vue-ui-socket">
+        <div class="vue-ui-socket"
+             for="${enc(attr: fieldid)}"
```

Comment:
For is an attribute reserved for labels, if you’d rather have all the data here then it’s better to change it to a data attribute, ie ‘data-field-id’